### ISSUE TITLE
Integrate iReport into the Galaxy history

### ISSUE THREAD

**Issue by bgruening** [shiltemann/iReport#1]
I really think such a project is missing in Galaxy and it would be worth to integrate it more deeply into Galaxy core. I imagine a history menu with an option 'create report' which will redirect you to a config page. 
Here you should have different plugins, like PDF, HTML, tgz, IPython Notebook?, Galaxy Page? etc ...

What do you think?

**Comment by hexylena** [comment]
I'd +1 that, as long as this tool can still be used as part of a workflow and preconfigured.

**Comment by shiltemann** [comment]
Yes, I was thinking of investigating something like that at the hackathon this summer. If we can make it so that it can also still be preconfigured as part of a workflow I think it would be great.
